evaluation/longbench/create_huggingface_dataset.py

```diff
@@ -0,0 +1,177 @@
+# SPDX-FileCopyrightText: Copyright (c) 1993-2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Could you add 1993-2025 (see also latest PR).
Please also include the header in `calculate_metrics.py`